back/src/domains/convention/adapters/pole-emploi-gateway/HttpPoleEmploiGateway.ts

```diff
@@ -145,7 +144,15 @@ export class HttpPoleEmploiGateway implements PoleEmploiGateway {
             originalId: poleEmploiConvention.originalId,
           },
         });
-        return { status: response.status as 200 | 201 };
+
+        return {
+          status: response.status,
+          ...([200, 201].includes(response.status)
+            ? {}
+            : {
+                feedback: { message: "Unsupported response status", response },
+              }),
+        } satisfies PoleEmploiBroadcastResponse;
       })
       .catch((err) => {
         const error = castError(err);
```

Comment:
Au lieu de mettre les satisfies à tous les returns, on peut faire ça :

```suggestion
      .catch((err): PoleEmploiBroadcastResponse => {
        const error = castError(err);
```